requirements.txt

```diff
@@ -1,5 +1,6 @@
-pyModbus
+pyModbus~=2.5.2
```

Comment:
I know it is not the proper way, but while the library is still in beta I prefer not to fix the versions of the requirements.
Remove the numbers